README.md

```diff
@@ -1,74 +1,23 @@
-![ddev logo](images/ddev_logo.png)
-
+# ddev
 [![CircleCI](https://circleci.com/gh/drud/ddev.svg?style=shield)](https://circleci.com/gh/drud/ddev) [![Go Report Card](https://goreportcard.com/badge/github.com/drud/ddev)](https://goreportcard.com/report/github.com/drud/ddev) ![project is maintained](https://img.shields.io/maintenance/yes/2017.svg)
 
-# ddev
+![ddev logo](images/ddev_logo.png)
 
 ddev is an open source tool that makes it dead simple to get local PHP development environments up and running within minutes. It's powerful and flexible as a result of its per-project environment configurations, which can be extended, version controlled, and shared. In short, ddev aims to allow development teams to use Docker in their workflow without the complexities of bespoke configuration.
 
-## Roadmap
-
-Each DRUD product has a dedicated product owner, who serves as the primary advocate for customers and end-users when making decisions regarding the public roadmap. For the [ddev roadmap](https://github.com/drud/ddev/wiki/roadmap), @rickmanelius is currently serving as the product owner.
-
-We use the longer-term roadmap to prioritize short-term sprints. Please review the [ddev roadmap](https://github.com/drud/ddev/wiki/roadmap) and [ddev issue queue](https://github.com/drud/ddev/issues) to see what's on the horizon.
+## Getting Started
 
-## System Requirements
+1. **Check System Requirements:** We support macOS High Sierra, Windows 10, and Linux ([more info here](https://ddev.readthedocs.io/en/latest/#system-requirements)).
+2. **Install ddev:** Options include [macOS homebrew](https://ddev.readthedocs.io/en/latest/#homebrew-macos) (recommended), an [install script](https://ddev.readthedocs.io/en/latest/#installation-script-linux-and-macos), or [a manually download](https://ddev.readthedocs.io/en/latest/#manual-installation-linux-and-macos).
+3. **Choose a CMS Quick Start Guide:** 
+  - [Wordpress](https://ddev.readthedocs.io/en/latest/users/cli-usage#wordpress-quickstart)
+  - [Drupal 7](https://ddev.readthedocs.io/en/latest/users/cli-usage#drupal-7-quickstart)
+  - [Drupal 8](https://ddev.readthedocs.io/en/latest/users/cli-usage#drupal-8-quickstart)
 
-- [Docker](https://www.docker.com/community-edition) version 17.05 or greater
-- OS Support
-  - macOS Sierra
-  - Linux
-    * Ubuntu 16.04 LTS
-    * Debian Jessie
-    * Fedora 25
-  - Windows 10 Pro
+Having trouble? See our [support options below](#support). Additionally, you may have trouble if [another local development tool is already using port 80 or 443](https://ddev.readthedocs.io/en/latest/#using-ddev-with-other-development-environments).
 
-We are open to expanding this list to include additional OSs as well as improve our existing support for the ones listed above. Please [let us know](https://github.com/drud/ddev/issues/new) if you hit an issue!
+## Current Feature List
```

Comment:
A rote list of ddev commands is not a feature list. I don't even think the list of ddev commands belongs on the readme.

Reply:
I agree with this. Usage information is a RTD thing, but I would definitely not be opposed to a bullet list of features (an executive summary type of thing).

Reply:
I agree with this. Usage information is a RTD thing, but I would definitely not be opposed to a bullet list of features (an executive summary type of thing).

Reply:
Same. I was only leaving it because it was what we pasted before. Happy to purge it.